### PR TITLE
Schedules

### PR DESCRIPTION
**Pull request by vycius**

### CODE REVIEW

**Comment by gediminasel** [comment]
👍

**Comment by kuprelis** [comment]
:+1:

File: app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleDayBanner.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Created by dekedro on 16.8.23.
+ */
+public class ScheduleDayBanner implements ScheduleItem {
+    private Date date;
+
+    private SimpleDateFormat dateFormater = new SimpleDateFormat("MM/dd/yyyy");
```
Comment (gediminasel) [inline app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleDayBanner.java:12]: Manau geriau mm-dd

File: app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleEvent.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * Created by dekedro on 16.8.22.
+ */
+public class ScheduleEvent implements ScheduleItem {
+    private long startTime;
+    private long endTime;
+    private String name;
+    private ArrayList<User> lecturers = new ArrayList<>();
+
+    private SimpleDateFormat dateFormater = new SimpleDateFormat("hh:mm");
```
Comment (gediminasel) [inline app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleEvent.java:18]: Gal HH:mm

File: app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    }
+
+    private void getScheduleEvents(final NMAService nmaService) {
+        nmaService.getEvents().enqueue(new Callback<List<ScheduleEvent>>() {
+            @Override
+            public void onResponse(Call<List<ScheduleEvent>> call, Response<List<ScheduleEvent>> response) {
+                List<ScheduleEvent> scheduleEvents = response.body();
+                setScheduleItems(scheduleEvents);
+                Toast.makeText(getContext(), "Size " + scheduleEvents.size(), Toast.LENGTH_SHORT).show();
```
Comment (gediminasel) [inline app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java:62]: Galutinei versijai nereikia

File: app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            }
+
+            @Override
+            public void onFailure(Call<List<ScheduleEvent>> call, Throwable t) {
+                Toast.makeText(getContext(), "Error" + t.getMessage(), Toast.LENGTH_SHORT).show();
```
Comment (gediminasel) [inline app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java:67]: Po error manau tarpo reikia

File: app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleDayBanner.java (hunk 1 above)
Comment (vycius) [reply app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleDayBanner.java:12]: Use `MM-dd`. https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

File: app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleEvent.java (hunk 2 above)
Comment (vycius) [reply app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleEvent.java:18]: Agree https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

File: app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleEvent.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    public String getLecturersNames() {
+        ArrayList<String> lecturersNames = new ArrayList<>();
+        for (User lecturer : lecturers) {
+            lecturersNames.add(lecturer.getName());
+        }
+        return TextUtils.join(", ", lecturersNames);
+    }
+
+    public Date getDate() {
+//        return new Date(1472023376l);
```
Comment (vycius) [inline app/src/main/java/com/nmakademija/nmaakademija/entity/ScheduleEvent.java:42]: Remove commented code

File: app/src/main/res/layout/schedule_event.xml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:padding="4dp">
+
+        <TextView
+            android:id="@+id/schedule_start_time"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:text="Hello World!" />
```
Comment (vycius) [inline app/src/main/res/layout/schedule_event.xml:17]: Use `tools:text` instead of `android:text`.

**Pull request closed without merging**